### ISSUE TITLE
Ctrl+Enter does not pass a new line to R when plotting device is opened

### ISSUE THREAD

**Issue by JanaJarecki** [randy3k/R-Box#34]
A bugreport:
After plotting, as long as the plotting device is opened, Crtl+Enter does not pass any lines of code on to R. This happens when I want to pass a single line to R while the plotting window is open. After I close the plotting window manually, passing code works like a charm.

Code to illustrate the issue: Note that the issue arises when passing the code line by line, not at once.
```R
    df = data.frame(x=c(1,1,2)) #will be executed
    plot(df) #will be executed
    # Now suppose I just typed this new line
    mean(df$x) #will not be executed
```

The issue occurred somwehen within the last version update(s).

Specs:
Windows 7, 64 bit, SublimeText 3, R-Box v2.3.3, using R in an external window.

**Comment by randy3k** [comment]
thanks for the report. I will fix it soon. Meanwhile, I believe if you change the focus of the window, the command should go to the console.

**Comment by randy3k** [comment]
fix by 4973fe1d169109f6a0aa0856c1c89ac24afe93b2

**Comment by JanaJarecki** [comment]
thanks!!
